ROMFS/px4fmu_common/init.d-posix/1061_tf-r1

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+#
+# @name ThunderFly TF-R1 UAV Rover
+# @type Rover
+# @class Rover
```

Comment:
Is this model also related to flight gear support?

Reply:
This model is compatible with PX4 rover support.